Fix for sizeof-pointer-memaccess error
libnetlink.c:45:23: error: argument to 'sizeof' in 'memset' call is the same expression as the destination; did you mean to dereference it? [-Werror=sizeof-pointer-memaccess]

--- a/libnetlink.c	2011-02-14 03:39:10.000000000 +0530
+++ b/libnetlink.c	2015-11-06 20:07:44.396021255 +0530
@@ -42,7 +42,7 @@
 	int sndbuf = 32768;
 	int rcvbuf = 32768;
 
-	memset(rth, 0, sizeof(rth));
+	memset(rth, 0, sizeof(*rth));
 
 	rth->fd = socket(AF_NETLINK, SOCK_RAW, protocol);
 	if (rth->fd < 0) {
